test/test_manifest.json

```diff
@@ -5800,6 +5800,12 @@
     "rounds": 1,
     "type": "eq"
   },
+    {  "id": "issue19532",
```

Comment:
Could you either indent correctly or run `npx gulp lint --fix` ?

Reply:
Will do.

Reply:
Also, please squash the commits.

Reply:
Into a single one, sure!

Reply:
I thought the `test/add_test.mjs` (https://github.com/mozilla/pdf.js/wiki/Contributing#4-run-lint-and-testing still says `.js` btw) would do the formatting correctly, but apparently not.